docs/user/automatic-redirects.rst

```diff
@@ -1,56 +1,140 @@
+
 For example::
 
     docs.readthedocs.io -> docs.readthedocs.io/en/latest/
     www.pip-installer.org -> www.pip-installer.org/en/latest/
 
-This only works for the root URL, not for internal pages. It's designed to redirect people from ``http://pip.readthedocs.io/`` to the default version of your documentation, since serving up a 404 here would be a pretty terrible user experience. (If your "develop" branch was designated as your default version, then it would redirect to ``http://pip.readthedocs.io/en/develop``.) But, it's not a universal redirecting solution. So, for example, a link to an internal page like ``http://pip.readthedocs.io/usage.html`` doesn't redirect to ``http://pip.readthedocs.io/en/latest/usage.html``.
+.. warning::
+
+   This only works for the root URL, not for internal pages.
+   It's designed to redirect people from ``http://pip.readthedocs.io/`` to the default version of your documentation,
+   since serving up a 404 here would be a pretty terrible user experience.
+
+.. note::
+   If the "develop" branch was designated as the default version,
+   then ``http://pip.readthedocs.io/`` would redirect to ``http://pip.readthedocs.io/en/develop``.
+   But, it's not a universal redirecting solution.
+   So, for example, a link to an internal page like
+   ``http://pip.readthedocs.io/usage.html`` doesn't redirect to ``http://pip.readthedocs.io/en/latest/usage.html``.
+
+   The reasoning behind this is that RTD organizes the URLs for docs so that multiple translations and multiple versions of your docs can be organized logically and consistently for all projects that RTD hosts.
+   For the way that RTD views docs,
+   ``http://pip.readthedocs.io/en/latest/`` is the root directory for your default documentation in English, not ``http://pip.readthedocs.io/``.
+   Just like ``http://pip.readthedocs.io/en/develop/`` is the root for your development documentation in English.
+
+Among all the multiple versions of docs,
+you can choose which is the "default" version for RTD to display,
+which usually corresponds to the git branch of the most recent official release from your project.
+
+Are "permalinks" possible?
+--------------------------
 
-The reasoning behind this is that RTD organizes the URLs for docs so that multiple translations and multiple versions of your docs can be organized logically and consistently for all projects that RTD hosts. For the way that RTD views docs, ``http://pip.readthedocs.io/en/latest/`` is the root directory for your default documentation in English, not ``http://pip.readthedocs.io/``. Just like ``http://pip.readthedocs.io/en/develop/`` is the root for your development documentation in English.
+You might be familiar with permalinks from blogging.
+The idea is that a blog post receives a unique link as soon as it's published and that the link does not change afterward.
+Incoming sources can reference the blog post even though the blog changes structure or the post title changes.
```

Comment:
I'm a bit reluctant to include it in our Glossary since we do not have _permalinks_ officially. I don't know if any documentation frameworks support this? I would love if they did, but I wanted to use the term so that people will understand that writing documentation and using a URL path is in some sense the same as creating a permalink.

Reply:
> I'm a bit reluctant to include it in our Glossary since we do not have permalinks officially 

Describing a term in the glossary does not mean that we officially support it. It helps us to expand the meaning of a word that in the same way through out the whole documentation, to give the user more context without extending the explanation of the word in all the pages.

In the glossary, we can explain things that we don't support at all; but we mention them in our documentation.

Reply:
Sounds good! It also aligns with an idea of being "friendly" to readers, i.e. make sure that they feel included when new terms are explained.